client/src/components/App/ColorPicker.js

```diff
@@ -73,7 +74,7 @@ class ColorPicker extends PureComponent {
                         onChangeComplete={this.handleColorChangeComplete}
                     />
                 </Popover>
-            </span>
+            </Fragment>
```

Comment:
(see previous comment)
```suggestion
            </>
```